.github/workflows/test.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           python-version: ${{ matrix.python-version }}
           architecture: x64
 
-      - run: pip install poetry
+      - run: pip install poetry==1.2.0a1
```

Comment:
Maybe use a build matrix for Poetry version?

Reply:
yup, that's an option, but I hope we can just update poetry before merging this PR, there's more dependencies that aren't ready for 3.10 :)